hookman/hookman_generator.py

```diff
@@ -140,11 +140,16 @@ def generate_plugin_template(self,
         Generate a template with the necessary files and structure to create a plugin
 
         A folder with the same name as the shared_lib_name argument will be created, with the following files:
-            - config.yml
-            - plugin.c
-            - hook_specs.h
-            - CMakeLists.txt
-            - README.md
+            <plugin_folder>
+                - CMakeLists.txt
+                - compile.py
+                <assets>
```

Comment:
Isn't this a fixed name, like `assets`? This this line should be `assets/`, same for `source_folder` below (`src/`)